packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -1514,6 +1524,15 @@ export default class RestApi {
         res.sendStatus(200)
       })
     )
+
+    api.delete(
+      '/groups/delete/:id',
+      wrap(async (req, res) => {
+        const { id } = req.params
+        await app.deleteGroup(id)
+        res.sendStatus(200)
```

Comment:
[204](https://developer.mozilla.org/fr/docs/Web/HTTP/Status/204) The action was successful and no content to return
```suggestion
        res.sendStatus(204)
```